R/cr_cn.r

```diff
@@ -104,6 +110,28 @@
 #'    style = "plant-cell-and-environment")
 #' cr_cn("10.5284/1011335", url = "https://data.datacite.org", 
 #'    style = "plant-cell-and-environment")
+#' 
+#' # with caching
+#' ## never expire
+#' webmiddens::wm_enable()
+#' webmiddens::wm_configuration("foostuff")
+#' cr_cn(dois="10.1126/science.169.3946.635", verbose=TRUE)
+#' cr_cn(dois="10.1002/app.27716", verbose=TRUE)
+#' 
+#' (dois <- cr_r(10))
+#' system.time((b=cr_cn(dois)))
+#' system.time((f=cr_cn(dois)))
+#' system.time((g=cr_cn(dois)))
+#' 
+#' ## expire
+#' Sys.setenv(WEBMIDDENS_EXPIRY_SEC = 5)
```

Comment:
why does this have to be general for all webmiddens stuff, why is it not an option for rcrossref alone? If I use several packages depending on webmiddens I might want a different expiry length?

Reply:
good idea to have sep env vars for each pkg. I like env vars because they are the easiest to use across computing environments, but i guess it'd make more sense to set expiry time via a function call if each pkg has their own specific expiration time